### PR TITLE
com.obsproject.Studio

### PR DESCRIPTION
**Pull request by TingPing**

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build com.obsproject.Studio

File: com.obsproject.Studio.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.obsproject.Studio.desktop</id>
+  <name>OBS Studio</name>
+  <developer_name>OBS Project</developer_name>
+  <summary>Free and open source software for video recording and live streaming</summary>
```
Comment (nedrichards) [inline com.obsproject.Studio.appdata.xml]: It'd be good to see a screenshot, longer description etc. here as well.

**Comment by nedrichards** [comment]
bot, build com.obsproject.Studio

**Comment by TingPing** [comment]
Will need to patch this out: https://github.com/jp9000/obs-studio/blob/a78daa8a1cafbbd81b6b44bd54989e0bd26e9882/deps/media-playback/CMakeLists.txt#L26-L30

**Comment by TingPing** [comment]
bot, build com.obsproject.Studio

**Comment by TingPing** [comment]
bot, build com.obsproject.Studio

**Comment by TingPing** [comment]
bot, build com.obsproject.Studio

**Comment by TingPing** [comment]
bot, build com.obsproject.Studio

**Comment by nedrichards** [comment]
bot, build com.obsproject.Studio

File: com.obsproject.Studio.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  "app-id": "com.obsproject.Studio",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "obs",
+  "rename-icon": "obs",
+  "rename-desktop-file": "obs.desktop",
+  "finish-args": [
+    "--socket=x11",
```
Comment (nedrichards) [inline com.obsproject.Studio.json]: KDE recommendation is now also to include wayland as I remember.

Comment (TingPing) [reply com.obsproject.Studio.json]: If you use Wayland OBS is less than useful considering it can't capture anything.

Comment (nedrichards) [reply com.obsproject.Studio.json]: Fair point

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/com.obsproject.Studio

**Pull request closed without merging**